### PR TITLE
Show and Tell 37

### PR DESCRIPTION
**Pull request by chrislo**

### CODE REVIEW

File: soups/show-and-tell/show-and-tell-37.snip.markdown [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+Before [Ben](ben-griffiths) sold an old laptop he thought he'd better have a look on the disk to see if there was anything valuable. He found a few old projects that he'd worked on while CTO at [MakieLab](http://mymakie.com/).
+
+These projects were part of the system which allowed users of the website to design, and then have 3D-printed, their own customised doll.
+
+![An early Makie prototype doll](/images/blog/makie-doll.jpg)
+
+The doll's face was a composite of more than 30 different models which each varied one parameter such as the size of the nose, the length of the ears, or the spacing between the eyes. By moving sliders on the web site users could blend between these different models to vary the face.
+
+At the time Ben built the system, the support for Adobe Flash was starting to wane. Ben used [WebGL]() to render the model in supported browsers and also developed a pure-JavaScript rendering engine as a fallback. Ben had unearthed one of the first versions of the code where he had succeeded in rendering a single eye ball.
```
Comment (chrisroos) [inline soups/show-and-tell/show-and-tell-37.snip.markdown]: Does this `[WebGL]()` link work? I thought you needed square brackets instead of parentheses.

Comment (floehopper) [reply soups/show-and-tell/show-and-tell-37.snip.markdown]: Fixed in ed56e28f62f51a57f73ccf49c48ba2bb502e81fa.

**Comment by floehopper** [comment]
@chrislo Is this still active or have you merged it manually?

**Comment by chrislo** [comment]
It's been merged manually. Closing.

**Pull request closed without merging**